package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>osqp-eigen</name>
+  <version>0.6.2</version>
+  <description>Simple Eigen-C++ wrapper for OSQP library </description>
+  <maintainer email="tbd@tbd.tbd">tbd</maintainer>
+
+  <license>LGPLv3</license>
+
+  <url type="website">https://github.com/robotology/osqp-eigen</url>
+
+  <build_depend>git</build_depend>
+  <!-- The following tags are recommended by REP-136 -->
+  <exec_depend condition="$ROS_VERSION == 1">catkin</exec_depend>
+  <depend>osqp</depend>
+  <depend>eigen</depend>
```

Comment:
Minor comment, not blocking the merge of anything, just a small curiosity. I checked out as I was not an expert, and according to https://www.ros.org/reps/rep-0149.html#depend-multiple the argument of a `depend` tag must be "rosdep key or ROS package name". While `eigen` is a rosdep key (see https://github.com/ros/rosdistro/blob/rolling/2021-12-09/rosdep/base.yaml#L878), `osqp` does not seem to be neither a rosdep key nor a ROS package name (there is only a `osqp_vendor` package https://github.com/ros/rosdistro/blob/d11a46f743c0b517f7a3250ac14ab6d789a3acd0/noetic/distribution.yaml#L5432).

Reply:
Good call. I am locally (and in CI / industrial_ci) adding osqp as a dependency to the upstream workspace via a rosinstall file (using https://github.com/ori-drs/osqp, which just adds the package.xml on top of the existing project). This way we can do dependency resolution in CI quite nicely - we haven't upstreamed it/released it to the buildfarm though

Reply:
I hadn't seen `osqp_vendor` before (it was released after I started work on this branch last year). However, just know when I tried to install it both on my machine and in a fresh Docker, it cannot be found.

Reply:
I guess that in this context any package you install in the catkin/colcon workspace can be considered a "ROS package", hence it works nice. Thanks!